master.yaml

```diff
@@ -523,6 +549,10 @@ Outputs:
         Description: The URL endpoint for the civic2018 service
         Value: !Join [ "/", [ !GetAtt ALB.Outputs.LoadBalancerUrl, "2018" ]]
 
+    Civic2019DisasterGameServiceUrl:
```

Comment:
FYI, I am 98% sure at this point that these Output values are unnecessary, and I'm considering an experiment where I eliminate some of them and verify that none of the underlying services break.  Doesn't hurt us to add this now, but figured I'd share some context that might not be widespread.